Gemfile

```diff
@@ -362,3 +362,5 @@ gem 'json-jwt', '~> 1.15'
 gem "json-schema", "~> 4.3"
 
 gem "csv"
+
+gem "async", "~> 1.32"
```

Comment:
the async gem looks well supported. I have no concerns with adding it to our Gemfile.